crates/depguard-app/README.md

```
@@ -1,96 +1,30 @@
-- `check-default-features-explicit`
-- `check-no-multiple-versions`
-- `check-optional-unused`
-- `check-yanked-versions`
-
-## Related Crates
-
-- [`depguard-cli`](../depguard-cli/) - CLI entry point that consumes this crate
-- [`depguard-domain`](../depguard-domain/) - Business logic layer
-- [`depguard-render`](../depguard-render/) - Output rendering
-- [`depguard-settings`](../depguard-settings/) - Configuration resolution
+## Problem
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)